Aait.web.g1.paulos dessie.navbar

PaulosNed opened the pull request:
I have created The navbar component and added it on the root's layout component so that it is visible across all pages. I have also configured the layout component to have a default font of poppins as all the pages are designed using that font except navbar and footer. I have also added additional configuration to the tailwind config and global.css files.




![Screenshot (10)](https://github.com/RealEskalate/2023-project-phase-starter-projects/assets/93246229/6097d8f8-57e1-4b80-8165-0888bc812889)

TsehaynehGetaneh reviewed (commented):
Looks good, but before I give you some comments and suggestions fix the conflicts and also remove the teams/page.tsx if you don't need it for the navbar.

kindly fix those issues and I'll come back to you.

PaulosNed commented:
> Looks good, but before I give you some comments and suggestions fix the conflicts and also remove the teams/page.tsx if you don't need it for the navbar.
> 
> kindly fix those issues and I'll come back to you.

I have fixed the conflit and deleted the teams page

jebessaD reviewed (approved):
LGTM
